.github/workflows/checks.yml

```diff
@@ -0,0 +1,19 @@
+name: Checks
+on:
```

Comment:
Shouldn't we check on push to main branch too?

Reply:
+1

Reply:
done